### ISSUE TITLE
bug: Searchbar styling is totally broken

### ISSUE THREAD

**Issue by bene-we** [ionic-team/ionic#18781]
# Bug Report

**Ionic version:**
[x] **5.1.0**

**Current behavior:**
I am using two different searchbars at the top of to different tab pages. When loading tab1 directly, searchbar1 shows as expected. If I navigate to tab2 with a tabbar, searchbar2 looks like this:

![image](https://user-images.githubusercontent.com/41572666/61140209-4ede0400-a4cb-11e9-9141-f98f8fa9a861.png)

When loading tab2 directly, everything's fine, but the same error appears when navigating to tab1.

**Expected behavior:**
Display the searchbar as usual:

![image](https://user-images.githubusercontent.com/41572666/61140407-b300c800-a4cb-11e9-88aa-489cf0b38624.png)

**Steps to reproduce:**
I am currently unable to test this in encapsulated pages. I can try to break it down in the next week. Nevertheless here is my code:

`news.page.html` (tab2)
```
<ion-header class="header-transparent">
    <ion-toolbar>
        <ion-searchbar #searchbar
                       class="header-search"
                       placeholder="Suche nach Artikeln"
                       showCancelButton
                       (ionCancel)="isSearchbarOpened = false"
                       (keyup)="doSearch(searchbar.value)">
        </ion-searchbar>
    </ion-toolbar>
</ion-header>

```
`global.scss`
```
ion-searchbar.header-search input {
    border-radius: 6px !important;
}

/* make toolbar transparant */
/* https://gist.github.com/paulstelzer/cca1aa1a8669e985dbfa0bfd262093eb */

.md {
    .header-transparent {
        ion-toolbar {
            --background: transparent;
            --ion-color-base: transparent !important;
        }

        ion-searchbar {
            padding-top: 7px !important;
        }
    }
}

/* Remove bottom border on md */
.header-transparent.header-md::after {
    background-image: none;
}

/* Remove bottom border on ios */
.header-transparent.header-ios ion-toolbar:last-child {
    --border-width: 0;
}

```

**Other information:**
The styling reappears when reloading the page manually (which is not possible inside of an app).

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 5.1.0 (C:\Users\Bene\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework               : @ionic/angular 4.6.0
   @angular-devkit/build-angular : 0.13.9
   @angular-devkit/schematics    : 7.3.9
   @angular/cli                  : 7.3.9
   @ionic/angular-toolkit        : 1.5.1

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : android 8.0.0
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 4.0.1, (and 5 other plugins)

Utility:

   cordova-res : not installed
   native-run  : 0.2.6

System:

   NodeJS : v10.15.3 (C:\Program Files\nodejs\node.exe)
   npm    : 6.4.1
   OS     : Windows 10

```

**Comment by bene-we** [comment]
![20190712_175930](https://user-images.githubusercontent.com/41572666/61141965-fc064b80-a4ce-11e9-811b-d85b9436129f.gif)

I recorded the behavior, the loading time is due to the screen recording software, normally everything is smooth.

**Comment by liamdebeasi** [comment]
This issue was fixed in Ionic 4.6.1. Please update to the latest version and let me know if you are still running into this issue.

Thanks!

**Comment by bene-we** [comment]
Updating took a while due to @angular/cli issues, but the bug is resolved now, thanks!
